terraform/variables.tf

```diff
@@ -13,8 +13,11 @@ variable "environment" {}
 
 variable "component_name" {}
 variable "dns_name" {}
+
 variable "s3_bucket_name" {}
 variable "s3_prev_bucket_name" {}
+variable "s3_backup_enabled" { default = false }
```

Comment:
We could do with a description here of what this means.

"Set to true if if the environment needs backing up e.g. in production. This will result in versioning and object locking being enabled for select buckets containing critical live data."